src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -568,7 +568,9 @@ private function doGet(string $id, int $invalidBehavior = ContainerInterface::EX
             $alias = $this->aliasDefinitions[$id];
 
             if ($alias->isDeprecated()) {
-                trigger_deprecation('', '', $alias->getDeprecationMessage($id));
+                $deprecation = $alias->getDeprecation($id);
+
+                trigger_deprecation($deprecation['package'], $deprecation['since_version'], $deprecation['message']);
```

Comment:
let's drop the keys and use `trigger_deprecation(...$alias->getDeprecation($id))` in all similar places, don't you think?